grails-app/services/org/broadinstitute/orsp/QueryService.groovy

```diff
@@ -786,22 +786,87 @@ class QueryService implements Status {
         result
     }
 
-    /**
-     * Find issues from a list of Ids
-     * @param issueIds
-     * @return
-     */
-    Set<Issue> findIssuesSearchItemsDTO(ArrayList<Integer> issueIds) {
-        final String query = "SELECT * FROM issue i WHERE i.id IN (:issueIds) and i.deleted = 0"
+/**
+ * Find issues and create DTOs to narrow the amount of data used in the search process
+ * @param issueIds
+ * @return
```

Comment:
nit: either fill out the `@return` or remove both `@`'s. I'm not worried about javadocs here, but we should be consistent.